cbrain_task/civet/views/_civet_file_from_col.html.erb

```diff
@@ -55,15 +55,10 @@
   <td><FONT COLOR="green">&#10004;</FONT></td>
 <% end %>
 
-<% if file[:t2_name].blank? && file[:pd_name].blank? %>
+<% if file[:t2_name].blank? || file[:pd_name].blank? %>
```

Comment:
The modifications to the view code is incorrect, the number of TD elements is not the same in the if and else cases, so the table is misaligned.

Reply:
See also the other file below.